### PR TITLE
Made scrolling after router-link not suck

### PR DESCRIPTION
**Pull request by lxgr-linux**
Here you go @Niko-Sn 

There seams to be a funny problem with firefox: https://github.com/vuejs/router/issues/1409

### CODE REVIEW

File: src/plugins/cardChain.js [hunk 1]
```diff
@@ -250,6 +250,44 @@ export default {
             throw new Error(err)
           })
       }
+      registerForCouncilTx () {
```
Comment (patrickwieth) [inline src/plugins/cardChain.js:253]: does not really belong to this PR's purpose, but still good to have it ^^

File: src/views/UserView.vue [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      <br>
+      Vote rights: {{ user.voteRights.length }}
+      <button
+        v-if="address == $store.getters['common/wallet/address']"
+        @click="$router.push({name: 'Vote'})"
+      >
+        Vote
+      </button>
+      <br>
+      Owned card schemes: {{ user.ownedCardSchemes.length }} <br>
```
Comment (patrickwieth) [inline src/views/UserView.vue:40]: we call this card frames now, card scheme is the old name which confused too many test users (In the blockchain we still call it that way, but this might also change over the long run)

**Comment by lxgr-linux** [comment]
Scrolling in firefox is now fixed https://github.com/vuejs/router/issues/1411
